### PR TITLE
Add support for COMMAND_STATUS_AUDIO in MPD server plugin; add ~#bitdepth and ~#samplerate

### PR DESCRIPTION
**Pull request by luk1337**

### CODE REVIEW

**Comment by luk1337** [comment]
Well, I thought that `-` wouldn't be that bad considering there's `~long-length` tag.

File: quodlibet/quodlibet/ext/events/mpdserver/main.py [hunk 1]
```diff
@@ -310,6 +310,10 @@ def status(self):
             state = "stop"
 
         status = [
+            ("audio", "%d:%d bps:%d" % (
```
Comment (lazka) [inline quodlibet/quodlibet/ext/events/mpdserver/main.py]: MPD seems to just do "%d:%d:%d": https://github.com/MusicPlayerDaemon/MPD/blob/44c60567dd7957b440c7643797bc223cf5aceb6e/src/AudioFormat.cxx#L59

Any reason for adding the "bps" bit?

Comment (luk1337) [reply quodlibet/quodlibet/ext/events/mpdserver/main.py]: I can drop bps bit.

File: quodlibet/quodlibet/ext/events/mpdserver/main.py [hunk 2]
```diff
@@ -310,6 +310,10 @@ def status(self):
...
+                info("~#sample_rate") or 0,
```
Comment (lazka) [inline quodlibet/quodlibet/ext/events/mpdserver/main.py]: ~#sample-rate, but wait until we have discussed the naming..

**Comment by lazka** [comment]
For comparison, beets has samplerate/bitdepth, foobar2000 has samplerate/__bitspersample

**Comment by lazka** [comment]
Any idea why the tests are failing?

**Comment by luk1337** [comment]
I actually don't know why they are failing. Also about the naming, it's fine for me either way I don't really have my preference here.

**Comment by luk1337** [comment]
I think tests should pass now.

**Comment by luk1337** [comment]
Yup, Linux tests are succeeding now ^.^

**Comment by lazka** [comment]
> Also about the naming, it's fine for me either way I don't really have my preference here.

ok, let's go with "samplerate" and "bitdepth" please.

> I think tests should pass now.

looks good

**Comment by luk1337** [comment]
done.

**Comment by lazka** [comment]
Thanks!

**Pull request merged**